### ISSUE TITLE
[Bug]: TestBaseCollectorReminderBackOff failled on Darwin/x86 occasionally

### ISSUE THREAD

**Issue by jensenojs** [matrixorigin/matrixone#7939]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 
- Hardware parameters:
- OS type:
- Others: https://github.com/matrixorigin/matrixone/actions/runs/4131749202/jobs/7139689505
```

### Actual Behavior

![image](https://user-images.githubusercontent.com/56761542/217756907-21eed6c3-d7fe-4104-a22a-93d30badbd3e.png)

### Additional information

ut link : https://github.com/matrixorigin/matrixone/actions/runs/4131749202/jobs/7139689505

**Comment by xzxiong** [comment]
logs_80076

**Comment by xzxiong** [comment]
dup as #8480, fixed in #8485
